src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -45,34 +47,40 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
                 </div>
 
                 <ReactTable
-                    className="-highlight -striped has-text-centered"
+                    className="-highlight -striped has-text-right"
                     defaultPageSize={this.props.districtResults.length}
                     pageSize={this.props.districtResults.length}
                     showPaginationBottom={false}
                     data={data}
                     {...norwegian}
                     columns={[
                         {
-                            Header: "Fylke",
+                            Header: <span className="is-pulled-left"> {"Fylke"}</span> ,
                             accessor: "name",
-                            minWidth: this.props.districtWidth * 10,
+                            width: this.props.districtWidth * 10,
+                            Cell: (row) => {
+                                return <span className="is-pulled-left" >{row.value}</span>;
+                            },
                             Footer: (
                                 <span>
-                                    <strong>Alle fylker</strong>
+                                    <strong className="is-pulled-left">Alle fylker</strong>
                                 </span>
                             ),
                         },
                         {
-                            Header: "Stemmer",
+                            Header: <span className="is-pulled-right"> {"Stemmer"}</span> ,
```

Comment:
Some strings, such as on line 66 you have just the text between the tags. But other places, like this line, you use brackets and quotes. We should only use one or the other, preferrably avoiding putting brackets and quotes everywhere, and not mix the two.

Reply:
Agree